GenderPayGap.WebUI/Controllers/ScopeController.cs

```diff
@@ -143,7 +143,7 @@ public IActionResult ConfirmInScopeAnswers(string encryptedOrganisationId, int r
             
             SendScopeChangeEmails(organisation, viewModel.ReportingYear, currentSnapshotDate, ScopeStatuses.InScope);
 
-            return RedirectToAction("ManageOrganisationGet", "ManageOrganisations", new { encryptedOrganisationId = encryptedOrganisationId });
+            return RedirectToAction("ScopeDeclared", "Organisation", new {id = encryptedOrganisationId});

```

Comment:
Hmm, why do we need a new action?

Reply:
The original journey didn't have this scope declared page, figured it was probably best to call the existing confirmation view in the Organisation area rather than add another new one in the Scope area. New action is because the existing view was only used as a return of the POST for the new org scope journey